squirrel_interface_emote/CMakeLists.txt

```diff
@@ -22,7 +23,7 @@ find_package(Boost REQUIRED COMPONENTS
 ## Declare things to be passed to dependent projects
 catkin_package(
   LIBRARIES squirrel_knowledge_base
-  CATKIN_DEPENDS roscpp rospy std_msgs rosplan_knowledge_msgs rosplan_dispatch_msgs squirrel_hri_msgs 
+  CATKIN_DEPENDS roscpp rospy std_msgs rosplan_knowledge_msgs rosplan_dispatch_msgs squirrel_expression squirrel_hri_msgs squirrel_vad_msgs
```

Comment:
please delete the catkin_depend to `squirrel_expression`, it is unnecessary squirrel_interface_emote is just publishing a message to the` /expression` topic, this doesn't require a build dependency.